AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Campaigns/CampaignDetailQueryHandler.cs

```diff
@@ -82,5 +90,13 @@ public CampaignDetailQueryHandler(AllReadyContext context)
 
             return result;
         }
+
+        private CampaignDetailViewModel.CampaignManagerInviteStatus GetCampaignManagerInviteStatus(CampaignManagerInvite campaignManagerInvite)
+        {
+            if (campaignManagerInvite.IsAccepted) return CampaignDetailViewModel.CampaignManagerInviteStatus.Accepted;
+            else if (campaignManagerInvite.IsPending) return CampaignDetailViewModel.CampaignManagerInviteStatus.Pending;
+            else if (campaignManagerInvite.IsRejected) return CampaignDetailViewModel.CampaignManagerInviteStatus.Rejected;
+            else  return CampaignDetailViewModel.CampaignManagerInviteStatus.Revoked;
+        }
```

Comment:
No it's via email, but when revoked the link should no longer be valid to accept the invite so no-ops if they click it. Not sure if that's specified somewhere/included already.